src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Figure implements Drawable, AreaCalculator {
+    protected String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    @Override
+    public double getArea() {
+        return 0;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("No figure to draw");
```

Comment:
The `draw` method is not implementing any specific drawing behavior, which is not useful. Each specific figure class should override this method to provide a meaningful implementation for drawing the figure.